### PR TITLE
Improved the look of ReceiptInput with bootstrap

### PR DESCRIPTION
**Pull request by livseibert5**
In this PR:
- ReceiptInput.js: added formatting with bootstrap, new look shown in picture
Screenshot 2020-07-09 at 10 48 01 AM

### CODE REVIEW

**Comment by livseibert5** [comment]
It seems like all my previous changes are showing up in this PR as well - ignore them they should be resolved once the previous PR is merged

File: src/main/java/com/google/edith/DealItem.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
+package com.google.edith;
+
+/**
```
Comment (malachibre) [inline src/main/java/com/google/edith/DealItem.java:3]: The first asterisk on the line that starts the javadoc comment should align with the other asterisks like:
```
/**
 * Text
 */
```

File: edith-step/src/components/ReceiptInput.js [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
 
+/**
+ * Displays form that takes in user input for a dynamic number
+ * of grocery items and their prices and quantities and adds them
+ * to a list.
+ */
 export default class ReceiptInput extends React.Component {
+  /**
+   * Constructor
+   * @param props Setup state.
```
Comment (corycook) [inline edith-step/src/components/ReceiptInput.js:12]: Lint error:

> Missing JSDoc parameter type for 'props'.eslint(valid-jsdoc)

Please fix.

File: edith-step/src/components/ReceiptInput.js [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
     this.state = {items: [], itemName: '', itemPrice: 0.0, itemQuantity: 1};
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
+  /**
+   * Send a post request to the receipt-data servlet
+   * with a grocery item in it every time a new item
+   * is added.
+   * @param e Submission event.
```
Comment (corycook) [inline edith-step/src/components/ReceiptInput.js:25]: Lint error:

> Missing JSDoc parameter type for 'e'.eslint(valid-jsdoc)

Please fix.

File: edith-step/src/components/ReceiptInput.js [hunk 4]
```diff
@@ -18,99 +33,133 @@ export default class ReceiptInput extends React.Component {
...
       itemName: '',
       itemPrice: 0.0,
-      itemQuantity: 1
+      itemQuantity: 1,
     }));
   }
 
+  /**
+   * Update state when form changed.
+   * @param e Change event.
```
Comment (corycook) [inline edith-step/src/components/ReceiptInput.js:61]: Lint error

> Missing JSDoc parameter type for 'e'.eslint(valid-jsdoc)

Please fix.

File: edith-step/src/components/ReceiptInput.js [hunk 5]
```diff
@@ -18,99 +33,133 @@ export default class ReceiptInput extends React.Component {
...
+   */
   handleChange(e) {
     const value = e.target.value;
     this.setState({
-      [e.target.name]: value
+      [e.target.name]: value,
     });
   }
 
+  /** Render grocery list form and items. */
```
Comment (corycook) [inline edith-step/src/components/ReceiptInput.js:70]: Lint error

> Missing JSDoc @return for function.eslint(valid-jsdoc)

Please fix.

File: edith-step/src/components/ReceiptInput.js [hunk 6]
```diff
@@ -18,99 +33,133 @@ export default class ReceiptInput extends React.Component {
...
-          </tr>
+    <div id="grocery-list">
+      <ul className="list-group col-lg-3">
+        <li className=
+          "h-50 list-group-item d-flex justify-content-between align-items-center">
+          <span className="col-lg-1">Item</span>
+          <span className="badge badge-pill col-lg-1">Price</span>
+          <span className="badge badge-pill col-lg-1">#</span>
+        </li>
+        {props.items.map((item) => (
```
Comment (corycook) [inline edith-step/src/components/ReceiptInput.js:147]: Lint error

> 'items' is missing in props validationeslint(react/prop-types)

Please fix.

See https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md#prevent-missing-props-validation-in-a-react-component-definition-reactprop-types for more information about the error.

Possible replacement:

```jsx
const GroceryList = createReactClass({
  propTypes: {
    items: PropTypes.arrayOf(PropTypes.object), // or more specific if possible
  },
  render() {
    const props = this.props; 
    ... 
  },
});
```

File: src/test/java/com/google/edith/UserInsightsTest.java [hunk 7]
```diff
@@ -243,4 +243,4 @@ private void setEntityProperties(Entity entity, double price,
     entity.setProperty("quantity", quantity);
     entity.setProperty("date", date);
   }
-}
+}
```
Comment (corycook) [inline src/test/java/com/google/edith/UserInsightsTest.java:246]: Nit: have an empty line at the end of the file

**Pull request merged**